metabase/metabase
# Updating Close Icon

### Description
Global update to the close icon, with a small resize to the remove button in `ChartSettingFieldPicker`

### How to verify
Check some places where the close icon is used and ensure that the close icon is rendering as expected. Some examples:
- Most modals have a close icon in the corner
- Dashboard Parameters
- Toaster notifications
- Notebook Steps
- Dashboard Table Click Behavior
- Dashboard Edit mode (removing a card)

### Demo
Before:
![image](https://user-images.githubusercontent.com/1328979/222266176-afb24fdc-30d9-410b-91f6-43f15add8ce4.png)

After:
![image](https://user-images.githubusercontent.com/1328979/222266286-0ae32f1e-9d80-41d4-8c65-fa1dd2de1b76.png)

### Tests
I'm unclear how to add programmatic tests for an icon change like this, but suggestions are welcome. Existing Percy tests may pick up on a few things, but I mostly relied on manual testing
